src/platforms/kotlin/index.mdx

```diff
@@ -5,4 +5,14 @@ Select your platform to get started:
 - [Java](/platforms/java)
 - [Android](/platforms/android)
 
+## Kotlin Extensions
+
+<Note>
+
+Support for Kotlin Coroutines is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
```

Comment:
this should be under the `## Kotlin Coroutines` section in the `src/platforms/common/enriching-events/scopes.mdx` file probably otherwise anyone accessing from there would not see it